README.md

```diff
@@ -1,11 +1,9 @@
-# TakeShape Starter Auth0 + Stripe
+# TakeShape Starter Auth0 + Shopify
```

Comment:
I recommend running through the readme later on with somebody else and take notes / make changes as they encounter issues. I did that a couple times on the Stripe starter and it was really helpful.

Reply:
Still haven't done this yet, but will after today.